include/aspect/simulator.h

```diff
@@ -505,12 +505,30 @@ namespace aspect
        * particular the mesh, the solution vectors, etc) whereas others can
        * either be re-generated (matrices, DoFHandler objects, etc) or are
        * read from the input parameter file. See the manual for more
-       * information.
+       * information. These set of files are permanently stored.
```

Comment:
These...are -> This...is

Also, please add the new sentence before "See the manual..."